### DISCUSSION TITLE
Package versioning

### DISCUSSION THREAD

**Discussion by ThomUK** [nhs-r-community/NHSRplotthedots#170] [General]
Today v0.1.0 of this package is on CRAN, and the github main branch has moved beyond that version to v0.1.0:9000. However, using this development version in production would be risky because of the possibility of accidental breaking changes.

What is the best versioning strategy for the minor fixes and feature additions that are being included periodically, before v0.2.0 goes to CRAN?

Could i propose that we start to use the 3rd digit of the version number for minor bugfix changes eg. v0.1.1

This might allow production users to take advantage of the latest changes without risking accidental breaking changes in production work.

**Comment by ChrisBeeley** [comment]
We do [semantic versioning](https://semver.org/) in our team, which I think is what you're referring to (?) and yes I agree it's a good way to go. I mean, I don't write any code in this repo, so what do I know, but that's my opinion anyway :stuck_out_tongue_closed_eyes:

**Reply by ThomUK** [reply]
Semantic versioning is what i was trying to remember, thanks!

**Reply by tomjemmett** [reply]
Worth having a quick catchup, I think we are probably at a point to call this 0.1.1 and get pushed to CRAN

**Reply by Lextuga007** [reply]
Could we make a catch up open to all? I don't think you'll be inundated with people  and this is a great question for package developers. I'd definitely like to come!

**Reply by ThomUK** [reply]
Definitely, how is Weds morning next week?

**Reply by tomjemmett** [reply]
I'm free anytime 8.30 - 11 :-)

**Comment by ThomUK** [comment]
There's a loose plan to meet on this next Weds - please dm me on slack to join or propose a better time...
